assets/src/edit-story/elements/video/frame.js

```diff
@@ -14,14 +14,127 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { CSSTransition } from 'react-transition-group';
+
 /**
  * Internal dependencies
  */
 import StoryPropTypes from '../../types';
 import MediaFrame from '../media/frame';
+import { elementFillContent } from '../shared';
+import { useStory } from '../../app/story';
+import { ReactComponent as PlayIcon } from '../../icons/play.svg';
+import { ReactComponent as PauseIcon } from '../../icons/pause.svg';
+
+const Play = styled(PlayIcon)`
+  width: 48px;
+  height: 48px;
+`;
+const Pause = styled(PauseIcon)`
+  width: 48px;
+  height: 48px;
+`;
+
+const Wrapper = styled.div`
+  ${elementFillContent}
+`;
+
+const ButtonWrapper = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  cursor: pointer;
+
+  opacity: 0;
+  &.button-enter {
+    opacity: 0;
+  }
+  &.button-enter-active,
+  &.button-enter-done {
+    opacity: 1;
+    transition: opacity 200ms;
+  }
+  &.button-exit {
+    opacity: 1;
+  }
+  &.button-exit-active,
+  &.button-exit-done {
+    opacity: 0;
+    transition: opacity 200ms;
+  }
+`;
 
 function VideoFrame({ element }) {
-  return <MediaFrame element={element} />;
+  const { id } = element;
+  const [hovering, setHovering] = useState(false);
+  const videoNode = document.getElementById(`video-${id}`);
```

Comment:
We could use `displayLayer.querySelector` here instead, to make sure not to find some other video instance (e.g. in carousel).

Reply:
Right, but it still won't be a valid approach with duplicated ids, so I think we should remove the root cause and remove ids from Carousel (or utilize classes in the worst case?).

We agreed yesterday on Slack, that there will be no video preview in Carousel thumbnails, so we don't need video elements there, so we can just put the poster image and save some resources?

Reply:
Solved in https://github.com/google/web-stories-wp/pull/720/commits/6c5214ade5c0d7007159226ed07a4baa4807e048